PubHub/PubHub.Common/Models/BookModel.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace PubHub.Common.Models
+{
+    public class BookModel
```

Comment:
As discussed the naming convetion for models are: [Entity][Function]Model

Example: UserUpdateModel